app/Policies/RelationPolicy.php

```diff
@@ -46,6 +46,11 @@ public function create(User $user)
      */
     public function update(User $user, Relation $relation)
     {
+        // If the relation is empty, this call is comming from the bulk delete check
```

Comment:
Not sure why this is needed? Updating already works. This code was added for delete where the model is empty